plugins/nodes/src/js/components/NodesTable.js

```diff
@@ -23,12 +23,15 @@ class NodesTable extends PureComponent {
     super();
     this.displayName = "NodesTable";
 
+    this.renderHealth = this.renderHealth.bind(this);
```

Comment:
Don't we normally have this `METHODS_TO_BIND` construct for that or is it only used some of the times?

Reply:
no idea 😅 I've seen in some files, but its missing in others. not sure what our current best practice is.